src/__tests__/commands/support/support.test.ts

```
@@ -0,0 +1,362 @@
+/**
+ * Test Suite: Support Command
+ *
+ * Comprehensive tests for the support command module.
+ * Tests cover command structure, execution logic, validation, modal creation,
+ * and error handling scenarios.
+ */
+
+import { describe, it, expect, beforeEach, vi } from 'vitest';
+import {
+	SlashCommandBuilder,
```

ESLint

[error] 'SlashCommandBuilder' is defined but never used.

(@typescript-eslint/no-unused-vars)